src/Modules/UngradedModules.jl

```diff
@@ -251,15 +382,18 @@ Return the (canonical) generators of the free module `F`.
 gens(F::AbstractFreeMod) = basis(F)
 
 @doc Markdown.doc"""
+    basis(F::AbstractFreeMod, i::Int)
+
     gen(F::AbstractFreeMod, i::Int)
 
-Return the `i`th generator of `F`, that is the `i`th unit vector.
+Return the `i`th basis vector of `F`, that is, return the `i`th unit vector.
```

Comment:
OK, by now I really think we are using different definitions for the term "unit vector". For me, and in all the books and lecture notes I have at hand, that refers to a vector of length 1 in a normed vector space. Are  you perhaps thinking of the German term "Einheitsbasis" which sometimes is used to refer to the "canonical basis" of a vector space K^n / free module R^n, whose members are the "kanonische Einheitsvektoren" / "canonical unit vectors" ?